conan/api/subapi/upload.py

```diff
@@ -120,17 +120,23 @@ def upload_backup_sources(self, files):
         for file in files:
             basename = os.path.basename(file)
             full_url = url + basename
+            is_summary = file.endswith(".json")
+            file_kind = "summary" if is_summary else "file"
             try:
-                # Always upload summary .json but only upload blob if it does not already exist
-                if file.endswith(".json") or not uploader.exists(full_url, auth=None):
-                    output.info(f"Uploading file '{basename}' to backup sources server")
+                if is_summary or not uploader.exists(full_url, auth=None):
```

Comment:
Do we want to gracefully compare the existing summary file in the remote, and use the comparison for the warning?